src/Mount.cpp

```diff
@@ -664,6 +702,81 @@ bool Mount::connectToDriver( TMC2209Stepper* driver, const char *driverKind ) {
 #endif
 #endif
 
+/////////////////////////////////
+//
+// configureFocusdriver
+// TMC2209 UART only
+/////////////////////////////////
+#if (FOCUS_STEPPER_TYPE != STEPPER_TYPE_NONE) && (FOCUS_DRIVER_TYPE == DRIVER_TYPE_TMC2209_UART)
+#if SW_SERIAL_UART == 0
+  void Mount::configureFocusDriver(Stream *serial, float rsense, byte driveraddress, int rmscurrent, int stallvalue)
+  {
+    _driverFocus = new TMC2209Stepper(serial, rsense, driveraddress);
+    _driverFocus->begin();
+    bool _UART_Rx_connected = false;
+    #if UART_CONNECTION_TEST_TXRX == 1
+        _UART_Rx_connected = connectToDriver( _driverFocus, "Focus" );
+        if (!_UART_Rx_connected) {
+            digitalWrite(ALT_EN_PIN, HIGH);    //Disable motor for safety reasons if UART connection fails to avoid operating at incorrect rms_current
+        }
+    #endif
+    _driverFocus->toff(0);
+    #if USE_VREF == 0  //By default, Vref is ignored when using UART to specify rms current.
+        _driverFocus->I_scale_analog(0);
+    #endif
+    LOGV2(DEBUG_STEPPERS, F("Mount: Requested Focus motor rms_current: %d mA"), rmscurrent);
+    _driverFocus->rms_current(rmscurrent, 0.1f); //holdMultiplier = 1 to set ihold = irun
```

Comment:
Is the `0.1f` meant to be different to the below `0.01f?

Reply:
No, good catch!